fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -1876,6 +1937,9 @@ protected void readFields(DataInput in) throws IOException {
             }
         }
         if (Env.getCurrentEnvJournalVersion() >= FeMetaVersion.VERSION_117) {
+            if (Config.isCloudMode()) {
+                cloudClusterId = Text.readString(in);
```

Comment:
@w41ter  pls take a look at this change

Reply:
It seems that this field was added after `VERSION_123` in selectdb. Please add another condition to check whether to read this field.

Reply:
add check version >= 123